src/plugins/cloudinary/index.js

```diff
@@ -255,9 +255,10 @@ class CloudinaryContext {
 
     const refresh = () => {
       const src = this.source();
-      const { posterColor } = this.player.cloudinary.posterOptions();
-      if (posterColor) {
-        this.disablePoster(posterColor);
+      const posterOptions = Object.assign({}, this.player.cloudinary.posterOptions(), src.getInitOptions().poster);
+
+      if (posterOptions.posterColor) {
+        this.disablePoster(posterOptions.posterColor);
```

Comment:
I am not sure i understand the logic here even before your change... like if posterColor is supplied as an option then we continue to disable it, why is that?

Reply:
instead of `disablePoster` you can call it `resetPosterAndSetPosterColorIfConfigured` :) 
I really don't like the whole poster flow, but like you said yesterday, we shouldn't/can't fix everything now